### PR TITLE
Improved usage instructions for navMenu hooks

### PR DESCRIPTION
**Pull request by michaelmcandrew**
See also https://github.com/civicrm/civicrm-dev-docs/pull/490

### CODE REVIEW

File: src/CRM/CivixBundle/Resources/views/Code/module.php.php [hunk 1]
```diff
@@ -143,11 +143,11 @@ function <?php echo $mainFile ?>_civicrm_preProcess($formName, &$form) {
...
-  _<?php echo $mainFile ?>_civix_insert_navigation_menu($menu, NULL, array(
-    'label' => E::ts('The Page'),
-    'name' => 'the_page',
-    'url' => 'civicrm/the-page',
-    'permission' => 'access CiviReport,access CiviContribute',
+  _<?php echo $mainFile ?>_civix_insert_navigation_menu($menu, 'Mailings', array(
+    'label' => E::ts('New subliminal message'),
+    'name' => 'mailing_subliminal_message',
+    'url' => 'civicrm/mailing/subliminal',
+    'permission' => 'access CiviMail  ',
```
Comment (totten) [inline src/CRM/CivixBundle/Resources/views/Code/module.php.php]: Extra spaces here?

File: src/CRM/CivixBundle/Resources/views/Code/module.civix.php.php [hunk 2]
```diff
@@ -355,8 +355,11 @@ function _<?php echo $mainFile ?>_civix_glob($pattern) {
  * Inserts a navigation menu item at a given place in the hierarchy.
  *
  * @param array $menu - menu hierarchy
- * @param string $path - path where insertion should happen (ie. Administer/System Settings)
- * @param array $item - menu you need to insert (parent/child attributes will be filled for you)
+ * @param string $path - path to where the item should be inserted. Made up of a
+ * series of 'name' attributes concatenated with /, e.g. 'Mailing',
+ * 'Administer/System Settings' or 'my_extension/submenu'
```
Comment (totten) [inline src/CRM/CivixBundle/Resources/views/Code/module.civix.php.php]: FWIW, the pre-existing code-style here is a bit off, e.g. we're supposed to look more like:

```
 *  @param string $path
 *     Path to where the item should be inserted. Made up of a series of 'name' 
 *     attributes concatenated...
```

Aside: Perhaps there's a simpler word to describe this? Like "the parent path"?

**Comment by michaelmcandrew** [comment]
@totten - a related issue: https://github.com/civicrm/civicrm-core/pull/11744. Depending on how quickly you think that one might get in, we can leave this PR open and add in some weights

**Comment by totten** [comment]
Thanks, Michael. It does look better. I'm inclined to go ahead and merge this one.

**Pull request merged**